zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java

```diff
@@ -137,17 +137,14 @@ public void remove(String noteId) throws IOException {
    * copy new/updated notes from source to destination storage
    * @throws IOException
    */
-  void sync(int sourceRepoIndex, int destRepoIndex) throws IOException {
+  public void sync(int sourceRepoIndex, int destRepoIndex) throws IOException {    
```

Comment:
Good point, thank you for notice! It was changed through the implementation, but is not required any more, so will revert. Reflected in the scope above.